### DISCUSSION TITLE
Limit number of sessions by user group

### DISCUSSION THREAD

**Discussion by clefss** [authelia/authelia#7712] [Ideas]
I'm not sure if this is just a feature that would be great for me, but it would be great if we could limit the number of sessions each user can have - configurable by group, for example. This would help prevent account sharing by allowing only one active login at a time, with the most recent session replacing any older ones.

**Comment by james-d-elliott** [comment]
Due to the severe security implications of this specifically the ability to cause a denial of service attack this isn't something we're interest in entertaining.
